llarp/nodedb.cpp

```diff
@@ -173,15 +173,15 @@ llarp_nodedb::Insert(const llarp::RouterContact &rc)
   return true;
 }
 
-ssize_t
+size_t
 llarp_nodedb::Load(const fs::path &path)
 {
   std::error_code ec;
   if(!fs::exists(path, ec))
   {
     return -1;
```

Comment:
this should be set to 0 if the return type is `size_t`

Reply:
I'm gonna revert this.